holidays/countries/antigua_and_barbuda.py

```
@@ -0,0 +1,123 @@
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+from holidays import JAN, MAR
+from holidays.groups import ChristianHolidays, InternationalHolidays, StaticHolidays
+from holidays.observed_holiday_base import (
+    ObservedHolidayBase,
+    SUN_TO_NEXT_MON,
+    SAT_SUN_TO_NEXT_MON_TUE,
+    SAT_SUN_TO_NEXT_MON,
+    MON_TO_NEXT_TUE,
```

Ruff (0.8.2)

`holidays.observed_holiday_base.MON_TO_NEXT_TUE` imported but unused

Remove unused import: `holidays.observed_holiday_base.MON_TO_NEXT_TUE`

(F401)